node/src/GlideClient.ts

```diff
@@ -534,11 +538,15 @@ export class GlideClient extends BaseClient {
      * ```
      */
     public async functionLoad(
-        libraryCode: string,
-        replace?: boolean,
-    ): Promise<string> {
+        libraryCode: GlideString,
+        options?: {
+            replace?: boolean;
+            decoder?: Decoder;
+        },
```

Comment:
```suggestion
        options?: { replace?: boolean } & DecoderOption
```
And update doc (add a list there)
Find an example of such type of option in #2160 and #2179